CMakeLists.txt

```diff
@@ -52,7 +53,7 @@ set(PRODUCT_SOURCE_REPOSITORY "<unspecified>" CACHE STRING "Product source branc
 set(PRODUCT_BITS "${BUILD_BITS}bit")
 
 set(PRODUCT_LIB_NAME "apache-geode" CACHE STRING "Binary name")
-set(PRODUCT_DLL_NAME "Apache.Geode" CACHE STRING ".Net Binary name")
+set(PRODUCT_DLL_NAME "${PRODUCT_VENDOR}.${PRODUCT_BASE_NAME}" CACHE STRING ".Net Binary name")
```

Comment:
Not sure we need this... given the line above was not changed..

Reply:
We don't specifically need this in CMakeLists.txt, but per the above we need PRODUCT_BASE_NAME in the assembly info, so may as well use it if we have it.